websites/S/Stremio/metadata.json

```diff
@@ -4,23 +4,39 @@
 		"name": "Dark_Ville",
 		"id": "638080361179512853"
 	},
+	"contributors": [
+		{
+			"name": "Sleeyax",
+			"id": "226037514954080257"
+		}
+	],
```

Comment:
Feel free to take over maintaining the presence.

Reply:
Thank you, I will!